internal/resources/examples_test.go

```diff
@@ -103,6 +103,13 @@ func TestAccExamples(t *testing.T) {
 				testutils.CheckCloudInstanceTestsEnabled(t)
 			},
 		},
+		{
+			category: "k6",
+			testCheck: func(t *testing.T, filename string) {
+				t.Skip() // TODO: Make all examples work
```

Comment:
All tests here are skipped, so I added the one for k6 examples but left it skipped as well.